profiles/models.py

```diff
@@ -51,12 +52,53 @@ def __ne__(self, other: Any) -> bool:
         return not self.__eq__(other)
 
 
+class Affiliation(db.Model):
+    _id = db.Column(db.Integer, name='id', primary_key=True)
```

Comment:
Nearest thing to an ID that the ORCID API appears to return is a `put-code`, so comparing values instead. Need to look more into it.